src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+| ------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| component     | REQUIRED              | string        | Component of the representational system described in `*_coordinatesystem.tsv` that the channel contains.                                                                                      |
+| name          | REQUIRED              | string        | Label of the channel. Entries have to match headers in (any) `*_motion.tsv.`                                                                                                                   |
+| tracked_point | REQUIRED              | string        | Label of the point that is being tracked, for example, label of a tracker or a marker (`LeftFoot`, `RightWrist`).                                                                              |
+| type          | REQUIRED              | string        | Type of data.                                                                                                                                                                                  |
+| units         | REQUIRED              | string        | Physical unit of the value represented in this channel, for example, vm for virtual meters, radian or degrees for angular quantities. See the BIDS spec for guidelines for Units and Prefixes. |
+
+SHOULD be present:
+
+| **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                                                                             |
+| ------------------ | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| placement          | RECOMMENDED           | string        | Placement of the tracked point on the body (for example, participant, avatar centroid, torso, left arm). It can refer to an external vocabulary for describing body parts.  |
+| description        | OPTIONAL              | string        | Brief free-text description of the channel, or other information of interest.                                                                                               |
+| sampling_frequency | OPTIONAL              | number        | Nominal sampling rate of the channel in Hz. In case the sampling rates differ between channels or from SamplingFrequency in *_motion.json file, this can be specified here. |
+| status_description | OPTIONAL              | string        | Brief free-text description of the channel, or other information of interest.                                                                                               |
+
```

Comment:
In MEG specs channel file, there's alst the `status` field, with options: "good", "bad", "n/a". Should it apply also here? Or shall we assume all channels provided are always good
```suggestion
| status  | OPTIONAL              | string        |  Data quality observed on the channel. A channel is considered bad if its data quality is compromised by excessive noise. If quality is unknown, then a value of n/a may be used. Description of noise type SHOULD be provided in [status_description].

Must be one of: "good", "bad", "n/a".    |
```